### DISCUSSION TITLE
Custom buttons?

### DISCUSSION THREAD

**Discussion by KyllianGamer** [JnCrMx/discord-game-sdk4j#51] [Q&A] [answered]
So I have seen these people have these custom buttons that for example say "Check out my YouTube" and then it actually links to the URL of their YouTube, stuff like that. How exactly does that work and can it be replicated with the sdk4j library?

**Comment by JnCrMx** [comment, marked as answer]
It is currently not supported by the GameSDK it seems: https://github.com/discord/gamesdk-and-dispatch/issues/123

**Reply by KyllianGamer** [reply]
Huh, well I have seen it, and am seeing it at this moment. Is it possible the text on the spectate button and a different button are able to be manipulated? All tough I also wouldn't know how the URLs work past that point.

Example of a custom RP with buttons: 
![afbeelding](https://user-images.githubusercontent.com/42489040/188284920-b23748e2-c22b-43ba-b170-0b5f6f66329e.png)

**Reply by KyllianGamer** [reply]
Oh wait I just got what you meant, the Game SDK doesn't support it but some other thing does. My bad!
